### PR TITLE
New Rule - Linux Auditd Hidden Files - Steganography

### PR DESCRIPTION
**Pull request by zakibro**
Rule for hidding zip files in images, was considering also logging for steghide - will try to do that later as it is not installed by default in linux

### CODE REVIEW

**Comment by frack113** [comment]
Hello 
As it is a global rule , please read  #1976 .
Do you have any references for the command line used ?
Thanks

**Comment by zakibro** [comment]
Would you recommend splitting this rule to two separate rules or trying to include the unzip command in this without using the global rule?

**Comment by frack113** [comment]
Hi,
simple way 2 files so SOC got 2 differents alerts .
for cat a title like 'Steganography Hide Zip Information in Picture File'
for zip a title like 'Steganography Unzip Hidden Information in Picture File'
Feel free to correct the english in title 😄

File: rules/linux/auditd/lnx_auditd_hidden_zip_files_steganography.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+logsource:
+   product: linux
+   service: auditd
+detection:
+   commands:
+       type: EXECVE
+       a0: cat
+   a1:
+       - a1|endswith: '.jpg'
+       - a1|endswith: '.png'
```
Comment (frack113) [inline rules/linux/auditd/lnx_auditd_hidden_zip_files_steganography.yml:25]: same detection but more readable 
```yaml
    a1|endswith:
        - '.jpg'
        - '.png'
 ```

File: rules/linux/auditd/lnx_auditd_unzip_hidden_zip_files_steganography.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+   service: auditd
+detection:
+   commands:
+       type: EXECVE
+       a0:
+           - unzip
+   a1:
+       - a1|endswith: '.jpg'
+       - a1|endswith: '.png'
+   condition: commands and a1
```
Comment (frack113) [inline rules/linux/auditd/lnx_auditd_unzip_hidden_zip_files_steganography.yml:27]: same detection but more readable 
```yaml
    a1|endswith:
        - '.jpg'
        - '.png'
 ```

**Pull request merged**